src/deterministic_vi_image_psf/elbo_image_psf.jl

```diff
@@ -313,3 +311,49 @@ function elbo_likelihood_with_fft!(
         accumulate_band_in_elbo!(ea, fsm_vec[n], sbs, gal_mcs, n, lanczos_width)
     end
 end
+
+
+function initialize_fft_elbo_parameters(
+    images::Vector{Image},
+    vp::VariationalParams{Float64},
+    patches::Matrix{SkyPatch},
+    active_sources::Vector{Int};
+    use_raw_psf=true)
+    
+    ea = ElboArgs(images, vp, patches, active_sources, psf_K=1);
+    load_active_pixels!(images, ea.patches; exclude_nan=false);
+    if use_raw_psf
+        psf_image_mat = Array{Matrix{Float64}}(ea.S, ea.N) 
+        for n in 1:ea.N, s in 1:ea.S
+            img = images[n]
+            world_loc = ea.vp[s][lidx.u]
+            pixel_loc = WCS.world_to_pix(img.wcs, world_loc)
+            psf_image_mat[s, n] = 
+                eval_psf(img.raw_psf_comp, pixel_loc[1], pixel_loc[2]);
+        end
+    else
+        psf_image_mat = Matrix{Float64}[
+            get_psf_at_point(ea.patches[s, b].psf) for s in 1:ea.S, b in 1:ea.N];        
+    end
+    fsm_vec = FSMSensitiveFloatMatrices[
+        FSMSensitiveFloatMatrices() for n in 1:ea.N];
+    initialize_fsm_sf_matrices!(fsm_vec, ea, psf_image_mat);
```

Comment:
Well, it was done but not pushed, so those semicolons are in master now.  :)  Hope that's ok.